audit/auditlogtextformatter.go

```diff
@@ -47,9 +48,13 @@ type AuditLogTextFormatter struct {
 }
 
 func logPackage(sb *strings.Builder, noColor bool, idx int, packageCount int, coordinate types.Coordinate) {
+	w := tabwriter.NewWriter(sb, 9, 3, 0, '\t', 0)
```

Comment:
You are using this `tabwriter` in three different places, and setting it up identically. I would probably just do the setup in `Format` and then pass a pointer to it to the methods that need it (sorta like `sb *strings.Builder` above). All of those functions are private so you can change the contract as much as you want!

Reply:
Oh sick! I was worried about whether I'd break something by doing that

Reply:
Naw, we (rather @zendern ) did a great job of making sure the right stuff was private so we could do whatever we wanted.